starter_project/autonomy/src/perception.hpp

```diff
@@ -45,6 +49,9 @@ namespace mrover {
         std::vector<StarterProjectTag> mTags;
         ros::Publisher mTagPublisher;
 
+        size_t image_width;
```

Comment:
I would instead modify `selectTag` to take in a `cv::Mat` instead of keeping track of global state